be/src/column/datum.h

```diff
@@ -129,9 +135,10 @@ class Datum {
     }
 
 private:
-    using Variant = std::variant<std::monostate, int8_t, uint8_t, int16_t, uint16_t, uint24_t, int32_t, uint32_t,
-                                 int64_t, uint64_t, int96_t, int128_t, Slice, decimal12_t, DecimalV2Value, float,
-                                 double, DatumArray, HyperLogLog*, BitmapValue*, PercentileValue*, JsonValue*>;
+    using Variant =
```

Comment:
comment  why we need both JsonValue and JsonValue* in datum.h

Reply:
done